iis-workflows/iis-workflows-transformers/src/test/java/eu/dnetlib/iis/workflows/transformers/spark/citationmatching/direct/SparkCitationMatchingDirectTransformerWorkflowTest.java

```diff
@@ -0,0 +1,22 @@
+package eu.dnetlib.iis.workflows.transformers.spark.citationmatching.direct;
+
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import eu.dnetlib.iis.IntegrationTest;
+import eu.dnetlib.iis.core.AbstractOozieWorkflowTestCase;
+
+/**
+ * 
+ * @author madryk
+ *
+ */
+@Category(IntegrationTest.class)
+public class SparkCitationMatchingDirectTransformerWorkflowTest extends AbstractOozieWorkflowTestCase {
+
+	@Test
+	public void testTransformer() {
+		
+		testWorkflow("eu/dnetlib/iis/workflows/transformers/spark/citationmatching/direct/with_sample_data");
```

Comment:
Well. I didn't check it personally, but i tried to stick with convensions used in iis before. I believe that there might be some problems with priming when there will be two workflow.xml files under the same location in classpath (one in /src/main/resources and second in /src/test/resources)